scvi/module/_multivae.py

```diff
@@ -0,0 +1,529 @@
+from typing import Dict, Iterable, Optional
+
+import numpy as np
+import torch
+from torch.distributions import Normal, Poisson
+from torch.distributions import kl_divergence as kld
+
+from scvi import _CONSTANTS
+from scvi._compat import Literal
+from scvi.distributions import NegativeBinomial, ZeroInflatedNegativeBinomial
+from scvi.module._peakvae import Decoder as DecoderPeakVI
+from scvi.module.base import BaseModuleClass, LossRecorder, auto_move_data
+from scvi.nn import DecoderSCVI, Encoder, FCLayers
+
+
+class LibrarySizeEncoder(torch.nn.Module):
+    def __init__(
+        self,
+        n_input: int,
+        n_cat_list: Iterable[int] = None,
+        n_layers: int = 2,
+        n_hidden: int = 128,
+        use_batch_norm: bool = False,
+        use_layer_norm: bool = True,
+        deep_inject_covariates: bool = False,
+    ):
+        super().__init__()
+        self.px_decoder = FCLayers(
+            n_in=n_input,
+            n_out=n_hidden,
+            n_cat_list=n_cat_list,
+            n_layers=n_layers,
+            n_hidden=n_hidden,
+            dropout_rate=0,
+            activation_fn=torch.nn.LeakyReLU,
+            use_batch_norm=use_batch_norm,
+            use_layer_norm=use_layer_norm,
+            inject_covariates=deep_inject_covariates,
+        )
+        self.output = torch.nn.Sequential(
+            torch.nn.Linear(n_hidden, 1), torch.nn.LeakyReLU()
+        )
+
+    def forward(self, x: torch.Tensor, *cat_list: int):
+        return self.output(self.px_decoder(x, *cat_list))
+
+
+class MULTIVAE(BaseModuleClass):
+    """
+    Variational auto-encoder model for joint paired + unpaired RNA-seq and ATAC-seq data.
+
+    Parameters
+    ----------
+    n_input_regions
+        Number of input regions.
+    n_input_genes
+        Number of input genes.
+    n_batch
+        Number of batches, if 0, no batch correction is performed.
+    n_labels
+        Number of labels, if 0, all cells are assumed to have the same label
+    gene_likelihood
```

Comment:
Seems like most of these have defaults, but only a few have them mentioned in the docstring. Is there a general convention on what has default indicated? (e.g. n_layers_encoder)

Reply:
As far as I can tell it's consistent with other models in the codebase